source/_docs/z-wave/entities.markdown

```diff
@@ -0,0 +1,179 @@
+---
+layout: page
+title: "Z-Wave Entity Naming"
+description: "A summary of common entity names."
+date: 2017-09-21 10:59
+sidebar: true
+comments: false
+sharing: true
+footer: true
+---
+
+<p class='note'>
+This is a work in progress, based upon reports in the forum, the author's own devices, and reading of various documentation. It will be incomplete, so if you have a device that isn't reported here, or have a device that reports a different value, please provide a report in the [Z-Wave section](https://community.home-assistant.io/c/configuration/zwave) of the forum or the #zwave channel on [Discord](https://discord.gg/RkajcgS). 
```

Comment:
If you want to have the complete spec you can find it here: http://zwavepublic.com/sites/default/files/SDS10242-29%20-%20Z-Wave%20Device%20Class%20Specification.pdf

Reply:
Thanks. I'll use that for the next round of updates

Reply:
Would be great if this doc was linked from the main zwave page.  I just found it after using zwave since day one and its a treasure trove of in information! Great job!